### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by plata**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  * Development
+  * Functions
+  * Games
+  * Graphics
+  * Internet
+  
+2. Create a new folder for your script inside that category. The folder structure should be:
+```
+category
+└── script-name
```
Comment (qparis) [inline README.md]: The structure is actually category -> application - script.
Here "Online" is the name of the script

File: README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  {
+		"scriptName": "Online",
+		"compatibleOperatingSystems": ["MACOSX", "LINUX"],
+		"testingOperatingSystems": [],
+		"free": true,
+		"requiresNoCD": false
+  }
+  ```
+  * main.png: application icon (400px x 300px)
+  * [application.json](https://github.com/plata/Scripts/wiki/application.json): describes the application
```
Comment (qparis) [inline README.md]: Please use the main git repos, I'll give you the access

Comment (plata) [inline README.md]: When I click Wiki, it loads https://github.com/PlayOnLinux/Scripts/wiki but never gets there. Instead, it stays on https://github.com/PlayOnLinux/Scripts.

Comment (qparis) [inline README.md]: I've created it and have given you the access. Should be good now

**Pull request merged**